src/reverb.js

```diff
@@ -103,20 +103,7 @@ class Reverb extends Effect {
    */
   process(src, seconds, decayRate, reverse) {
     src.connect(this.input);
-    var rebuild = false;
-    if (seconds) {
-      this._seconds = seconds;
-      rebuild = true;
-    }
-    if (decayRate) {
-      this._decay = decayRate;
-    }
-    if (reverse) {
-      this._reverse = reverse;
-    }
-    if (rebuild) {
-      this._buildImpulse();
-    }
+    this.set(seconds, decayRate, reverse);
```

Comment:
I'm curious what bug this fixes? It would be helpful to see the corresponding test that helped you identify this as a solution.

In any case, good call to re-use the method rather than duplicate code!

Reply:
Yeah, Sorry for that. It does not fix any bug. Just rearranged the code